yt_dlp/extractor/xvideos.py

```diff
@@ -157,3 +157,24 @@ def _real_extract(self, url):
             'thumbnails': thumbnails,
             'age_limit': 18,
         }
+
+
+class XVideosQuickiesIE(InfoExtractor):
+    IE_NAME = 'xvideos:quickies'
+    _VALID_URL = r'https?://(?P<domain>(?:[^/]+\.)?xvideos2?\.com)/amateur-channels/[^#]+#quickies/a/(?P<id>\w+)'
```

Comment:
I gave `\w+` in example code since idk how the ids can be. But looking at above extractor, it should be `\d+`?

Reply:
```suggestion
    _VALID_URL = r'https?://(?P<domain>(?:[^/]+\.)?xvideos2?\.com)/amateur-channels/[^#]+#quickies/a/(?P<id>\d+)'
```